setup.cfg

```diff
@@ -39,6 +39,7 @@ warn_redundant_casts = True
 warn_unused_ignores = True
 disallow_untyped_calls = True
 disallow_incomplete_defs = True
+disallow_untyped_defs = True
```

Comment:
what about the other `disallow_xy` from the issue? I think it was about decorators?